feat: read config json from documents dir

Saschl opened the pull request:
<!-- Add the issues this PR fixes here. If no issues are related to this PR, then this line can be removed. -->
<!-- Add further issues with a full "Fixes #[issue_no]" line to ensure GitHub closes each one when the PR is merged. -->
Needs https://github.com/flybywiresim/simbridge/pull/121 to be released first.

## Summary of Changes
<!-- Please provide a summary of changes for this pull request, ensuring all changes are explained. -->

## Screenshots (if necessary)
<!-- If your PR includes visual changes, screenshots from before and after your change should always be included. -->
<!-- Please make best efforts to provide useful before and after screenshots. They should match camera angle, zoom, size, time of day, etc. -->

## Additional context
<!-- Add any other context about the pull request here. -->

<!-- You may optionally provide your discord username, so that we may contact you directly about the issue. -->
Discord username (if different from GitHub):

FoxtrotSierra6829 commented on src/renderer/utils/InstallerConfiguration.ts:
```diff
@@ -354,7 +354,7 @@ export interface Configuration {
 export class InstallerConfiguration {
   static async obtain(): Promise<Configuration> {
-    return this.fetchConfigurationFromCdn()
+    return this.loadConfigurationFromLocalStorage()
```
please wait for #477 to implement a developer option to force the use of a local configuration or use https://github.com/flybywiresim/installer-data/pull/40 for testing.

Saschl replied:
Whoops that change wasn’t meant to be committed, will revert.

FoxtrotSierra6829 reviewed (changes requested):
![grafik](https://github.com/user-attachments/assets/18da193d-4155-451d-b752-2a9d68b7e76d)
In the about page, the html parsing is not implemented. Also, the fallback is missing for the resources folder which would be helpful to access to move the files before updating.

Saschl commented:
> ![grafik](https://private-user-images.githubusercontent.com/50082450/374955386-18da193d-4155-451d-b752-2a9d68b7e76d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc3NDEsIm5iZiI6MTcyODQ3NzQ0MSwicGF0aCI6Ii81MDA4MjQ1MC8zNzQ5NTUzODYtMThkYTE5M2QtNDE1NS00NTFkLWI3NTItMmE5ZDY4YjdlNzZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDEyMzcyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhZWZkNmUyODVkODEwYmI2MGViYTUzMjg4NDYxMmQyOWVkN2U4NGEzNTYwZjYxNGFlNjUwZWQ0ODc3ZGVmYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LjrBsfnh1uo-2yl50j4jE6oa_vx85748nUcJM6aUwwg) In the about page, the html parsing is not implemented. Also, the fallback is missing for the resources folder which would be helpful to access to move the files before updating.

Fixed the first, thanks 👍  About the fallback: we could also split the installer-data PR into two.

1. Only update the description to raise awareness -> installer works as before 
2. Update the path together with the simbridge release -> installer will open the new directory with the resources button

Saschl commented:
> Open for discussion, I'd rather prefer a solution that is not strictly time-based since not everyone uses the FlightSim on a daily basis

Yep, I see the point. I just want to keep the change as minimal as possible. Please check again, I've added another directories entries in the config which points to the old one. For me it's a good compromise :D 

Otherwise I would think it's too much. If we go with my original proposal, only people who ignore the update will have an issue with the "Resources" button and the configuration in the installer. I would consider that acceptable.
